api/lib/searchOperators.js

```diff
@@ -8,6 +10,27 @@ const constantsObj = {
   DecSearchOperators: [43, 45, 42, 34, 39],
 };
 
+// Pairing the bindi characters with their non-bind counterparts
+const bindiCharsUni = {
+  ਸ: 'ਸ਼',
+  ਖ: 'ਖ਼',
+  ਗ: 'ਗ਼',
+  ਜ: 'ਜ਼',
+  ਫ: 'ਫ਼',
+};
+
+// Generating an object with the ASCII codes of the bindi characters
+const bindiCharacters = Object.entries(bindiCharsUni).reduce((acc, [key, value]) => {
+  const char = anvaad.unicode(key, true);
+  const asciiCode = anvaad.ascii(char).replaceAll(',', '');
```

Comment:
can you shed some light ono why we need to use repalceAll here?

Reply:
Do we need to reduce this or should it be map?

Reply:
Removed this entire logic, and created an object directly.